packages/dd-trace/src/llmobs/channels.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { channel } = require('dc-polyfill')
+
+module.exports = {
```

Comment:
The main idea of channels is to not need to depend on shared objects directly like this and use names instead. What is the benefit of centrally defining channel objects?

Reply:
I think I was thinking it was more organized this way, but I see your point. But tbh I only use these channels in `index.js` where I subscribe/unsubscribe from everything, so I'll just move them there 👍